src/lib/components/table.svelte

```diff
@@ -87,6 +108,13 @@
                 <Paginator
                     onPageChange={(page) => {
                         currentPage = page
+
+                        if (data.rows.length / currentPage <= DEFAULT_TABLE_PAGE_SIZE) {
+                            showLoadMoreButton = true
```

Comment:
there is a weird behavior here because the loadMore button will keep appearing if you reached the end of the search results that will be provided by is, but you keep clicking on the last page. Repro steps:
- Go to a manager
- Keep loading more and more until you reach the end and no more results will be loaded
- If you click on the last page again, it will show the button load more again because the button always displays `onPageChange`

Reply:
Yes indeed. I currently see it more as a feature instead of a bug otherwise you won't be able to request newly added channels/identities added by others besides triggering the complete search again.